lib/stencil-init.js

```diff
@@ -1,18 +1,17 @@
 'use strict';
-var Fs = require('fs');
-var jsonLint = require('./json-lint');
-var Path = require('path');
-var Inquirer = require('inquirer');
-var themePath = process.cwd();
-var hoek = require('hoek');
-var configuration;
-var internals = {};
+const Fs = require('fs');
+const { promisify } = require("util");
+const Path = require('path');
+const Inquirer = require('inquirer');
+const hoek = require('hoek');
 
-internals.parseAnswers = function(JspmAssembler, ThemeConfig, dotStencilFile, dotStencilFilePath, answers) {
+const jsonLint = require('./json-lint');
+const themePath = process.cwd();
 
+async function performAnswers(JspmAssembler, ThemeConfig, stencilConfig, dotStencilFilePath, answers) {
```

Comment:
should this be `parseAnswers`

Reply:
@junedkazi  I decided to rename the function since it's actually _performing_ already _parsed_ answers :)